### ISSUE TITLE
Quest - Morbent Fel

### ISSUE THREAD

**Issue by NickEast** [scotty0100/wowascension-issues#244]
Name: Nickthemage
Account Name (for vote points rewards): NickEast
Map & Area: Duskwood
How to replicate the bug: Use quest item "Morbemt's Bane http://db.vanillagaming.org/?item=7297" on elite npc "Morbent Fel".
What happens: Npc enters combat mode and attacks player.
What should happen: Mob must turn from elite npc to normal npc with lower health and damage attributes, so a one player could kill him solo.
Located on WoWHead Map (if applicable):http://www.wowhead.com/npc=43761#drops
Source (proof, if applicable): 

Extra Information:

**Comment by Skylark14** [comment]
Did you equip the item first before trying to use it?

**Comment by NickEast** [comment]
Yup. Sure, tried to use the equipped item on that npc, as described above.
